crates/wasm-smith/src/core.rs

```diff
@@ -586,20 +586,21 @@ impl Module {
         index
     }
 
-    fn arbitrary_rec_group(&mut self, u: &mut Unstructured) -> Result<()> {
+    fn arbitrary_rec_group(&mut self, u: &mut Unstructured, allow_empty: bool) -> Result<()> {
```

Comment:
Can we create an `enum` with better names? The bool params are always hard to read at call sites.

Reply:
How would you feel `let allow_empty = false;` at callsites? I also feel like it's a pretty big burden to create a brand new enum every time a `bool` is passed...

Reply:
This one is passed around in a bunch of places so, while I definitely understand the drawback, it feels like it is amortized in this case. Not in the public api so shouldn't need docs either.

```rust
enum AllowEmptyRecGroup {
    Yes,
    No,
}
```